app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java

```diff
@@ -195,7 +207,16 @@ private void loadFragment(int navItemId) {
                 fragment = EventDashboardFragment.newInstance(eventId);
                 break;
         }
-        fragmentManager.beginTransaction().replace(R.id.fragment_container, fragment).commit();
+
+        getSupportFragmentManager().popBackStack();
```

Comment:
We want to pop it always?

Reply:
We're adding to backstack in all cases except DashboardFragment. So I think, yes.